### PR TITLE
Bugfix/v2.1.2 debug issues

### PR DESCRIPTION
**Pull request by kasuntech**

### CODE REVIEW

File: src/app/dashboard/home/home.component.ts [hunk 1]
```diff
@@ -255,10 +259,15 @@ export class HomeComponent implements OnInit, OnDestroy, AfterViewChecked, After
...
-    this.rucherService.rucher.photo = base64;
-    this.rucherService.updateBackgroundApiary(this.rucherService.rucher._id);
-    this.selectPhotoApiary = null;
+    if (base64.length * (3/4) < this.maxUploadFileSize) {
+      this.rucherService.rucher.photo = base64;
+      this.rucherService.updateBackgroundApiary(this.rucherService.rucher._id);
+      this.selectPhotoApiary = null;
+    } else {
+      this.notifyService.notify('warning', 'Le fichier est trop volumineux pour être téléchargé. (La taille maximale du fichier est de 6 Mo)');
+    }
```
Comment (lzpons) [inline src/app/dashboard/home/home.component.ts:268]: Good for the message,
one question : How are you managing this translation in EN for example?

File: src/main.ts [hunk 2]
```diff
@@ -19,6 +19,10 @@ import { environment } from './environments/environment';
 
 if (environment.production) {
   enableProdMode();
+
+  if (window) {
+    window.console.log = function() {};
+  }
```
Comment (lzpons) [inline src/main.ts:25]: What is this for?

File: src/app/dashboard/service/api/dailyRecordService.ts [hunk 3]
```diff
@@ -383,37 +383,37 @@ export class DailyRecordService {
...
-     * 
-     * @param hiveId 
-     * @param range 
+     *
+     * @param hiveId
+     * @param range
      */
     public getBroodByHive(hiveId: string, range: Date[]): Observable<any[]> {
-        return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/brood/' + hiveId, range);
+        return this.http.post<any[]>(CONFIG.URL + 'dailyRecordsTH/brood/old/' + hiveId, range);
```
Comment (lzpons) [inline src/app/dashboard/service/api/dailyRecordService.ts:410]: What does this /old/ mean?

File: src/app/dashboard/home/home.component.ts (hunk 1 above)
Comment (kasuntech) [reply src/app/dashboard/home/home.component.ts:268]: Just draft message in EN and translate it to  french  via google translator. I think, it is correct.

File: src/main.ts (hunk 2 above)
Comment (kasuntech) [reply src/main.ts:25]: for the avoid console log printing in production mode.

File: src/app/dashboard/service/api/dailyRecordService.ts (hunk 3 above)
Comment (kasuntech) [reply src/app/dashboard/service/api/dailyRecordService.ts:410]: server response 404 for this request 'dailyRecordsTH/brood/'.  this service REST URL is not found in back end. I checked your last deployed version 'https://bzz.mellisphera.com/', it use that  'dailyRecordsTH/brood/old/' URL. it is working properly. I think small code issue  from last version.

File: src/app/dashboard/home/home.component.ts (hunk 1 above)
Comment (lzpons) [reply src/app/dashboard/home/home.component.ts:268]: But I mean that depending on user prefs for lang message should display in EN, FR or ES

File: src/main.ts (hunk 2 above)
Comment (lzpons) [reply src/main.ts:25]: is there an option for debug or you simply removed the printout?

File: src/app/dashboard/service/api/dailyRecordService.ts (hunk 3 above)
Comment (lzpons) [reply src/app/dashboard/service/api/dailyRecordService.ts:410]: That is weird, 
looking to the database the *dailyRecordsTH* collection has a *brood* item that seems the target of this request

Capture d’écran 2020-07-21 à 10 36 34

we need to understand further why it appears this way.

File: src/app/dashboard/home/home.component.ts (hunk 1 above)
Comment (lzpons) [reply src/app/dashboard/home/home.component.ts:268]: you have translations in src/assets/i18n/en.json fr.json and es.json

**Comment by lzpons** [comment]
Hi Kasuntech, 
thanks for this request,
It would be preferable to merge your *bugfix/v2.1.2_debug_issues* branch with branch *feature/v2.1.2xxxx* instead of *dev* branch.

Can you edit your PR in that sense please?

**Comment by lzpons** [comment]
This PR is cancelled

**Pull request closed without merging**